pkg/trace/obfuscate/sql.go

```diff
@@ -342,3 +342,23 @@ func (o *Obfuscator) obfuscateSQL(span *pb.Span) {
 	}
 	traceutil.SetMeta(span, sqlQueryTag, oq.Query)
 }
+
+// ObfuscateSQLStringSafe obfuscates the sql string, returning either the successfully obfuscated string or a string
+// containing an error message explaining why the obfuscation failed
+func (o *Obfuscator) ObfuscateSQLStringSafe(sqlString string) string {
+	result, err := o.ObfuscateSQLString(sqlString)
+	if err != nil {
+		log.Debugf("failed to obfuscate sql string '%s': %s", sqlString, err.Error())
```

Comment:
```suggestion
		log.Debugf("Failed to obfuscate SQL string '%s': %s", sqlString, err.Error())
```
We had a PR a while back which made all log entries capitalised.

Reply:
Updated